### PR TITLE
Add delete data feature

### PR DESCRIPTION
**Pull request by myagley**

### CODE REVIEW

File: tempodb/client.py [hunk 1]
```diff
@@ -174,6 +184,15 @@ def _read(self, series_type, series_val, start, end, interval="", function="", t
...
         return DataSet.from_json(json)
 
+    def _delete(self, series_type, series_val, start, end):
+        params = {
+            'start': start.isoformat(),
+            'end': end.isoformat(),
+        }
+        url = '/series/%s/%s/data/' % (series_type, series_val)
+        json = self.request(url, method='DELETE', params=params)
+        return json
```
Comment (plainlystated) [inline tempodb/client.py:195]: is json a dictionary here?

File: tempodb/client.py [hunk 2]
```diff
@@ -110,6 +110,16 @@ def read_key(self, series_key, start, end, interval="", function="", tz=""):
         series_val = series_key
         return self._read(series_type, series_val, start, end, interval, function, tz)
 
+    def delete_id(self, series_id, start, end):
```
Comment (plainlystated) [inline tempodb/client.py]: Should we do the same strategy as we did for ruby (unused options dictionary, for future use)?

Comment (myagley) [inline tempodb/client.py]: Yeah, i can add options to the requests. We probably should

File: tempodb/client.py (hunk 1 above)
Comment (myagley) [inline tempodb/client.py:195]: No, it's a string. This needs to be addressed when we look at how to handle the responses better. That will get addressed with error handling code we discussed.

Comment (myagley) [inline tempodb/client.py:195]: In the meantime, I tried to match the conventions from the rest of the calls

Comment (plainlystated) [inline tempodb/client.py:195]: makes sense to me

**Comment by myagley** [comment]
Alright, added kwargs to the delete methods. I'll update the other methods when we touch them next

**Comment by plainlystated** [comment]
lgtm

**Pull request merged**